inc/api.class.php

```diff
@@ -936,6 +938,11 @@ protected function getItems($itemtype, $params = array(), &$totalcount=0) {
       $where = Search::addDefaultWhere($itemtype);
       if ($where == '') $where = "1=1 ";
 
+      // manage return of deleted elements
+      if ($item->maybeDeleted()) {
+         $where.= "AND `$table`.`is_deleted` = ".intval($params['is_deleted']);
```

Comment:
I checked if a whitespace is needed at the end of this line. It seems not. Maybe we can prepend a white space here and remove the trailing one in L937 / 939. This is for code consistency; I think this code will run fine anyway.